Add dev blog about backup agents (#2568)
home-assistant/developers.home-assistant: blog/2025-02-27-backup-agents.md

---
author: Martin Hjelmare
authorURL: https://github.com/MartinHjelmare

> title: "Backup agents"

Author: ```suggestion
title: Backup agents
```

---

> In the January release of 2025 we introduced a new feature for backups called backup agents.

Author: ```suggestion
In the January release of 2025, we introduced a new feature for backups called backup agents.
```

> Previously backup platforms were only used to pause or prepare integration operations before creating a backup and/or run some operation after a backup. With the introduction of backup agents, the backup platform now allows integrations to add one or more backup agents that can upload backups to some local or remote location.

Author: ```suggestion
Previously, backup platforms were used only to pause or prepare integration operations before creating a backup and to execute a post-backup operation. With the introduction of backup agents, the backup platform now allows integrations to add one or more backup agents that can upload backups to some local or remote location.
```
